File: Week 6/mario.py
from cs50 import get_int

# mak a nested loop to get a number between 0 and 9
while True:
    height = get_int("Height: ")
    if height > 0 and height < 9:
        break
# print the shape
for i in range(height):
    for j in range(height):
        if i + j >= height - 1:
            print("#", end="")
        else:
            print(" ", end="")
    print("")

File: Week 1/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    // Asking user for height and check if the height is between 0 and 8
    int height;
    do
    {
        height = get_int("Height: ");
    }
    while (height < 1 || height > 8);
    // print the shape of mario
    for (int i = 0 ; i < height ; i++)
    {
        for (int j = 0 ; j < height ; j++)
        {
            if (i + j >= height - 1)
            {
                printf("#");
            }
            else
            {
                printf(" ");
            }
        }
        printf("\n");
    }
}